test/SparqlAntlrParserTest.cpp

```diff
@@ -44,9 +44,9 @@ TEST(SparqlParser, NumericLiterals) {
   testNumericLiteral("3.0", 3.0);
   testNumericLiteral("3.0e2", 300.0);
   testNumericLiteral("3.0e-2", 0.030);
-  testNumericLiteral("3", 3ull);
+  testNumericLiteral("3", (int64_t)3ll);
   testNumericLiteral("-3.0", -3.0);
-  testNumericLiteral("-3", -3ll);
+  testNumericLiteral("-3", (int64_t)-3ll);
```

Comment:
I think the only three tests that are missing for complete coverage are the explicit positive numbers, so
`+3, +3.02 and +3.1234e12`
and the cases for "Decimals without a digit before the comma":
`.234, +.0123, -.5123`. 

And similar cases for doubles

`.234e4, +.0123E-3, -.5123E12`. 

then we also have the capital E for the exponent tested.

I think this should basically suffice, we are not exhaustively testing `stod`.

EDIT:

Maybe we should... stod and stoll can throw exceptions if the numbers are too big, and those should be caught with a reasonable message, and this should be tested!!!
Glad that I write such long comments, it gives me time to reason.